<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagnosticsDialog</class>
 <widget class="QDialog" name="DiagnosticsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diagnostics</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="subs_tab">
      <attribute name="title">
       <string>Scrobbling</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>-1</number>
         </property>
         <item>
          <widget class="StatusLight" name="subs_light" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="subs_status">
           <property name="toolTip">
            <string>This is an easter egg!</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>2</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="subs_cache_count">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="cached">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <column>
          <property name="text">
           <string>Artist</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Track</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Album</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fingerprintingTab">
      <attribute name="title">
       <string>Fingerprinting</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="fingerprints_title">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Recently Fingerprinted Tracks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="fingerprints">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <column>
          <property name="text">
           <string>Artist</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Track</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Album</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="iPodTab">
      <attribute name="title">
       <string>iPod Scrobbling</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QComboBox" name="ipod_type">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>iTunes automatically manages my iPod</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>I manually manage my iPod</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ipod_scrobble_button">
           <property name="text">
            <string>Scrobble iPod</string>
           </property>
           <property name="autoRepeatDelay">
            <number>300</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="ipod_log">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="maximumBlockCount">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="logs_button">
       <property name="text">
        <string>Logs</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string notr="true">&lt;a href='http://status.last.fm'&gt;http://status.last.fm&lt;/a&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_button">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatusLight</class>
   <extends>QWidget</extends>
   <header>lib/unicorn/widgets/StatusLight.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>DiagnosticsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
